images.go

```diff
@@ -0,0 +1,41 @@
+package main
+
+import (
+	"github.com/bwmarrin/discordgo"
+)
+
+
+
+func init() {
+	registerCommand("erasmus", "links", "Link naar erasmus", sayErasmus)
+}
+
+func sayErasmus(s *discordgo.Session, m *discordgo.MessageCreate) {
+	embed := NewEmbed()
+	embed.SetTitle("Erasmus @ ITfactory")
+	embed.SetImage("https://static.eyskens.me/thomas-bot/sem_2_2020.gif")
+	embed.SetURL("https://thomasmore365.sharepoint.com/sites/james/NL/international?tmbaseCampus=Geel")
+	s.ChannelMessageSendEmbed(m.ChannelID, embed.MessageEmbed)
+}
+
+func init() {
+	registerCommand("partners", "links", "Link naar partners", sayPartners)
+}
+
+func sayPartners(s *discordgo.Session, m *discordgo.MessageCreate) {
+	embed := NewEmbed()
+	embed.SetTitle("Partners in education")
+	embed.SetImage("https://static.eyskens.me/thomas-bot/voorstelling_partners_in_education.png")
+	s.ChannelMessageSendEmbed(m.ChannelID, embed.MessageEmbed)
+}
+
+func init() {
+	registerCommand("love", "fun", "Toon wat liefde aan elkaar <3", sayLove)
```

Comment:
een `init` functie kan je geen meerdere keren herhalen in een bestand, Je kan ze beter samenvoegen in de 1ste init
```go
func init() {
	registerCommand("erasmus", "links", "Link naar erasmus", sayErasmus)
        registerCommand("partners", "links", "Link naar partners", sayPartners)
        registerCommand("love", "fun", "Toon wat liefde aan elkaar <3", sayLove)
}
```

De `init` functie in go is een speciale functie, die is namelijk de 1ste functie die ge-called word als het programma het bestand laad. (in een compiled language weleenswaar...)

Reply:
Oh, fix incoming